plugins/check_curl.c

```diff
@@ -160,10 +160,16 @@ char *http_method = NULL;
 char *http_post_data = NULL;
 char *http_content_type = NULL;
 CURL *curl;
+int curl_global_initialized = 0;
```

Comment:
Could we use `bool` here to, well, tell readers and the compiler that it is really a boolean?

Reply:
Sure, can change that, it' is logically a bool (so are many more variables in check_curl.c). But check_curl.c was on purpose looking similar to check_http.c, so that backports (if necessary) would be easier.

Reply:
I will happily change all the pig booleans to real ones in `check_http` :-)

Reply:
`monitoring-plugins` doesn't compile with anything lower than C99 anyways (which is partially my fault :-) ). That's said and done. Given enough time and motivation I would like to change that everywhere.

Reply:
Ok, then it's no problem, will change.. :-)

Reply:
Great, thanks!